tests/plots/test_image.py

```diff
@@ -14,3 +14,26 @@ def test_random_multi_image():
     """
 
     shap.image_plot([np.random.randn(3, 20,20) for i in range(3)], np.random.randn(3, 20,20), show=False)
+
+def test_image_to_text_single():
+    """ Just make sure the image_to_text function doesn't crash.
+    """
+    
+    class MockImageExplaination:
```

Comment:
nit: MockImageExplaination -> MockImageExplanation

Reply:
Corrected the typo